src/lib.rs

```diff
@@ -327,7 +327,7 @@ impl PrivatePkcs1KeyDer<'_> {
 #[cfg(feature = "alloc")]
 impl zeroize::Zeroize for PrivatePkcs1KeyDer<'static> {
     fn zeroize(&mut self) {
-        self.0 .0.zeroize()
```

Comment:
I'm happy to see this get fixed. The old way is so ugly!

Reply:
me too! totally grot